portafly/.env

```diff
@@ -0,0 +1 @@
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
Please avoid committing user configurable dotfiles if they are not meant to be used for the project

Reply:
It was intended, more info is in my commit message: https://github.com/3scale/porta/pull/1626/commits/aafae3618b5fe45fe4b5011fa4183e8c9d93ef4b

Reply:
Really ... 🤦‍♂ This is 🐄 💩 https://github.com/facebook/create-react-app/issues/2560

Can't they use proper configuration ?!?  using environment even in build time is not good at all!
Imagine I have some other shell scripts that customises some of those environment or prevent them to be overridden, because why not I can ... Imagine the nightmare for debugging in development?

Why does React do that and not webpack for example? Something fishy with their answer.

Relying on committed environment does not mean it will act as expected, because environment are User controlled